website/docs/docs/advanced-usage/components.md

```diff
@@ -153,6 +153,31 @@ If you played with the `Sandpack` preset, you should be familiar already with th
   </SandpackLayout>
 </SandpackProvider>
 
+### Extensions
+
+Sandpack uses CodeMirror behind the hood to provide a nice editor. You can extend the editor with your own extensions, like [`@codemirror/autocomplete`](https://www.npmjs.com/package/@codemirror/autocomplete).
```

Comment:
`behind the scenes` or `under the hood` :) Also, I guess there isn't a nice list of codemirror extensions anywhere? I looked for it, but I couldn't find it. If you do know of one, a link to such a list might be nice?

```suggestion
Sandpack uses CodeMirror under the hood to provide a nice editor. You can extend the editor with any CodeMirror extensions, such as [`@codemirror/autocomplete`](https://www.npmjs.com/package/@codemirror/autocomplete).
```

Reply:
Oh, thanks for clarifying it to me! 

Yeah, I also haven't found any interesting extension list, that's why I pointed to an official one that I used in the examples.